en/advanced/commercial-use

```diff
@@ -0,0 +1,13 @@
+# Commercial use
+
+We believe that software should be accessible to everyone, which is why we offer JabRef completely free of charge. JabRef is available under an open-source license, which means that anyone can use, modify, and distribute it without restriction. We believe that this is the best way to promote innovation and collaboration.
+
+However, we also understand that some users may want to support our project financially, which is why we offer a "pay what you want" model for all commercial uses. This means that users can choose to donate any amount they want to support our project, while still being able to use our software for commercial purposes. We believe that this model is a flexible and transparent way to generate revenue while still allowing users to choose how much they want to pay.
+
+We want to be clear that JabRef is completely free to use for all commercial purposes, regardless of whether or not a user chooses to donate. We believe that open-source software should be accessible to everyone, regardless of their financial situation. Our goal is to provide high-quality, open-source software that can be used by anyone, anywhere in the world.
+
+We also want to emphasize that any donations we receive will be used to support the ongoing development and maintenance of our software. This includes things like server costs, bug fixes, and new feature development. We believe that by supporting our project financially, users can help to ensure that our software remains high-quality and up-to-date.
```

Comment:
We probably should adapt this part here. All donations are used to support the association purposes according to german law. This includes development of the software, maintenance cost, as well as supporting teaching of software engineering.

Remember the saying: For free you have received, for free you shall give. :-)